sampleSpecs/pizza.smithy

```diff
@@ -116,6 +116,12 @@ structure FallbackError {
   error: String
 }
 
+@error("client")
+structure FallbackError2 {
+  @required
+  error: String
+}
+
```

Comment:
Adding this was the only way to keep a current PizzaClientSpec test passing. We could also remove the test if we prefer. But in order to get an `UnknownErrorResponse` now, there must be two errors on an operation that have no status code with only a `@error("client")` annotation. Before this PR, it would never fallback to errors with no status code which is why the test case used to pass.

Reply:
This is perfect for me

maybe a comment on top of the shape that explains it serves as a way to add ambiguity to the error picking algorithm so that it falls back to `UnknownError` ?

Reply:
Yeah good idea, will add!